Tests fail on Windows

Running the tests via `gradlew check` on windows produces the following error.

```
org.springframework.jdbc.datasource.init.ScriptStatementFailedException: Failed to execute SQL script statement #1 of class path resource [de/budgetfreak/budgetfreakapplication/user/users.sql]: INSERT INTO BF_USER (NAME, CURRENCY) VALUES ('Max', 'â‚¬'); nested exception is org.h2.jdbc.JdbcSQLException: Wert zu gross / lang für Feld "CURRENCY VARCHAR(1) NOT NULL": "STRINGDECODE('\u00e2\u201a\u00ac') (3)"
Value too long for column "CURRENCY VARCHAR(1) NOT NULL": "STRINGDECODE('\u00e2\u201a\u00ac') (3)"; SQL statement:
INSERT INTO BF_USER (NAME, CURRENCY) VALUES ('Max', 'â‚¬') [22001-192]
	at org.springframework.jdbc.datasource.init.ScriptUtils.executeSqlScript(ScriptUtils.java:491)
	at org.springframework.jdbc.datasource.init.ResourceDatabasePopulator.populate(ResourceDatabasePopulator.java:238)
	at org.springframework.jdbc.datasource.init.DatabasePopulatorUtils.execute(DatabasePopulatorUtils.java:48)
	at org.springframework.jdbc.datasource.init.ResourceDatabasePopulator.execute(ResourceDatabasePopulator.java:253)
	at org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener$1.doInTransactionWithoutResult(SqlScriptsTestExecutionListener.java:237)
	at org.springframework.transaction.support.TransactionCallbackWithoutResult.doInTransaction(TransactionCallbackWithoutResult.java:34)
	at org.springframework.transaction.support.TransactionTemplate.execute(TransactionTemplate.java:133)
	at org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener.executeSqlScripts(SqlScriptsTestExecutionListener.java:234)
	at org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener.executeSqlScripts(SqlScriptsTestExecutionListener.java:142)
	at org.springframework.test.context.jdbc.SqlScriptsTestExecutionListener.beforeTestMethod(SqlScriptsTestExecutionListener.java:112)
	at org.springframework.test.context.TestContextManager.beforeTestMethod(TestContextManager.java:269)
	at org.springframework.test.context.junit4.statements.RunBeforeTestMethodCallbacks.evaluate(RunBeforeTestMethodCallbacks.java:74)
	at org.springframework.test.context.junit4.statements.RunAfterTestMethodCallbacks.evaluate(RunAfterTestMethodCallbacks.java:86)
	at org.springframework.test.context.junit4.statements.SpringRepeat.evaluate(SpringRepeat.java:84)
	at org.junit.runners.ParentRunner.runLeaf(ParentRunner.java:325)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:252)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.runChild(SpringJUnit4ClassRunner.java:94)
	at org.junit.runners.ParentRunner$3.run(ParentRunner.java:290)
	at org.junit.runners.ParentRunner$1.schedule(ParentRunner.java:71)
	at org.junit.runners.ParentRunner.runChildren(ParentRunner.java:288)
	at org.junit.runners.ParentRunner.access$000(ParentRunner.java:58)
	at org.junit.runners.ParentRunner$2.evaluate(ParentRunner.java:268)
	at org.springframework.test.context.junit4.statements.RunBeforeTestClassCallbacks.evaluate(RunBeforeTestClassCallbacks.java:61)
	at org.springframework.test.context.junit4.statements.RunAfterTestClassCallbacks.evaluate(RunAfterTestClassCallbacks.java:70)
	at org.junit.runners.ParentRunner.run(ParentRunner.java:363)
	at org.springframework.test.context.junit4.SpringJUnit4ClassRunner.run(SpringJUnit4ClassRunner.java:191)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.runTestClass(JUnitTestClassExecuter.java:114)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassExecuter.execute(JUnitTestClassExecuter.java:57)
	at org.gradle.api.internal.tasks.testing.junit.JUnitTestClassProcessor.processTestClass(JUnitTestClassProcessor.java:66)
	at org.gradle.api.internal.tasks.testing.SuiteTestClassProcessor.processTestClass(SuiteTestClassProcessor.java:51)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.dispatch.ContextClassLoaderDispatch.dispatch(ContextClassLoaderDispatch.java:32)
	at org.gradle.internal.dispatch.ProxyDispatchAdapter$DispatchingInvocationHandler.invoke(ProxyDispatchAdapter.java:93)
	at com.sun.proxy.$Proxy1.processTestClass(Unknown Source)
	at org.gradle.api.internal.tasks.testing.worker.TestWorker.processTestClass(TestWorker.java:108)
	at sun.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at sun.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at sun.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.lang.reflect.Method.invoke(Method.java:498)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:35)
	at org.gradle.internal.dispatch.ReflectionDispatch.dispatch(ReflectionDispatch.java:24)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:146)
	at org.gradle.internal.remote.internal.hub.MessageHubBackedObjectConnection$DispatchWrapper.dispatch(MessageHubBackedObjectConnection.java:128)
	at org.gradle.internal.remote.internal.hub.MessageHub$Handler.run(MessageHub.java:404)
	at org.gradle.internal.concurrent.ExecutorPolicy$CatchAndRecordFailures.onExecute(ExecutorPolicy.java:63)
	at org.gradle.internal.concurrent.ManagedExecutorImpl$1.run(ManagedExecutorImpl.java:46)
	at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1142)
	at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:617)
	at org.gradle.internal.concurrent.ThreadFactoryImpl$ManagedThreadRunnable.run(ThreadFactoryImpl.java:55)
	at java.lang.Thread.run(Thread.java:748)
Caused by: org.h2.jdbc.JdbcSQLException: Wert zu gross / lang für Feld "CURRENCY VARCHAR(1) NOT NULL": "STRINGDECODE('\u00e2\u201a\u00ac') (3)"
Value too long for column "CURRENCY VARCHAR(1) NOT NULL": "STRINGDECODE('\u00e2\u201a\u00ac') (3)"; SQL statement:
INSERT INTO BF_USER (NAME, CURRENCY) VALUES ('Max', 'â‚¬') [22001-192]
	at org.h2.message.DbException.getJdbcSQLException(DbException.java:345)
	at org.h2.message.DbException.get(DbException.java:179)
	at org.h2.table.Column.validateConvertUpdateSequence(Column.java:335)
	at org.h2.table.Table.validateConvertUpdateSequence(Table.java:784)
	at org.h2.command.dml.Insert.insertRows(Insert.java:151)
	at org.h2.command.dml.Insert.update(Insert.java:114)
	at org.h2.command.CommandContainer.update(CommandContainer.java:98)
	at org.h2.command.Command.executeUpdate(Command.java:258)
	at org.h2.jdbc.JdbcStatement.executeInternal(JdbcStatement.java:184)
	at org.h2.jdbc.JdbcStatement.execute(JdbcStatement.java:158)
	at org.springframework.jdbc.datasource.init.ScriptUtils.executeSqlScript(ScriptUtils.java:470)
	... 54 more
```

--- a/server/src/test/java/de/budgetfreak/budgetfreakapplication/DatabaseTest.java
+++ b/server/src/test/java/de/budgetfreak/budgetfreakapplication/DatabaseTest.java
@@ -6,11 +6,13 @@ import org.springframework.boot.test.context.SpringBootTest;
 import org.springframework.test.context.ActiveProfiles;
 import org.springframework.test.context.ContextConfiguration;
 import org.springframework.test.context.TestExecutionListeners;
+import org.springframework.test.context.jdbc.SqlConfig;
 import org.springframework.test.context.junit4.SpringRunner;
 
 @RunWith(SpringRunner.class)
 @ContextConfiguration(classes = BudgetfreakapplicationApplication.class)
 @ActiveProfiles("test")
 @DataJpaTest
+@SqlConfig(encoding = "UTF-8")
 public abstract class DatabaseTest {
 }

--- a/server/src/test/java/de/budgetfreak/budgetfreakapplication/user/UserRepositoryTest.java
+++ b/server/src/test/java/de/budgetfreak/budgetfreakapplication/user/UserRepositoryTest.java
@@ -8,24 +8,24 @@ import org.springframework.test.context.jdbc.Sql;
 
 import static org.assertj.core.api.Assertions.assertThat;
 
-@Sql("users.sql")
+@Sql(value = "users.sql")
 public class UserRepositoryTest extends DatabaseTest {
 
     @Autowired
     private UserRepository testSubject;
 
     @Test
-    public void shouldBeInitialized() throws Exception {
+    public void shouldBeInitialized() {
         assertThat(testSubject).isNotNull();
     }
 
     @Test
-    public void shouldNotBeEmpty() throws Exception {
+    public void shouldNotBeEmpty() {
         assertThat(testSubject.findAll()).hasSize(2);
     }
 
     @Test
-    public void maxShouldBeHere() throws Exception {
+    public void maxShouldBeHere() {
         final User max = testSubject.findByName("Max");
         assertThat(max).isNotNull();
         final SoftAssertions softly = new SoftAssertions();
@@ -35,7 +35,7 @@ public class UserRepositoryTest extends DatabaseTest {
     }
 
     @Test
-    public void mariaShouldBeHere() throws Exception {
+    public void mariaShouldBeHere() {
         final User maria = testSubject.findByName("Maria");
         assertThat(maria).isNotNull();
         final SoftAssertions softly = new SoftAssertions();
@@ -45,7 +45,7 @@ public class UserRepositoryTest extends DatabaseTest {
     }
 
     @Test
-    public void shouldSaveNewEntites() throws Exception {
+    public void shouldSaveNewEntites() {
         final User manuel = new User("Manuel", "€");
         testSubject.saveAndFlush(manuel);
         assertThat(testSubject.findAll()).hasSize(3);

